### DISCUSSION TITLE
zh/config/side

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#351] [Announcements]
# zh/config/side

我们在主页右下角能看到一个 图标,如果你电脑分辨率很高 可能图标比较小 展开后分别是 刷新 ; 新建文件; 新建文件夹; 聚合移动; 移除空的子目录; 批量重命名; 上传; 离线下载; 是否展开大纲; 开关复选框; 本地设置; 下面简单介绍一下大概说明吧，一些常见功能 如果你没有找到此功能，请更新版本 1. 刷新：我们常说的刷新不是使用F5刷新网页，而...

https://alist.nn.ci/zh/config/side.html

**Comment by Zding89** [comment]
请问列表项目文件名过长有办法默认设为多行显示吗?

**Comment by jiahong19887** [comment]
將以外
